src/test/java/com/github/fppt/jedismock/comparisontests/SimpleOperationsTest.java

```diff
@@ -492,6 +492,61 @@ public void scanIterates(Jedis jedis) {
         assertNotEquals(ScanParams.SCAN_POINTER_START, result.getStringCursor());
     }
 
+    public void sscanReturnsAllValues(Jedis jedis) {
```

Comment:
This is missing the `@Theory` annotation so i don't think it's running.

Reply:
nice catch, i added it and works fine